CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
 
 * **[Fix]** Fix application version being reported for ClickOnce deployments.
 
+### App Center Push
```

Comment:
I think version should be bumped

Reply:
Actually missing (Under development) for 2.4.0, it was not released.